CSharpTemplate.cs

```diff
@@ -8,14 +8,13 @@ namespace CommBank.Api.Provisioning.Services.Source.Templates
 {
     public class CSharpTemplate : ISourceTemplate
     {
-        private static readonly string[] DefaultIgnoreDirs = { ".git", "node_modules", "bower_components", "packages", "bin", "obj" };
-
         private readonly string _name;
```

Comment:
provide encapsulation for each field. assigning directly to filed is not advisable